JIB does not work

After ./gradlew --stop:

Caused by: java.lang.NoSuchFieldError: INSTANCE
        at org.apache.http.conn.ssl.SSLConnectionSocketFactory.<clinit>(SSLConnectionSocketFactory.java:151)

        at com.google.api.client.http.apache.v2.ApacheHttpTransport.newDefaultHttpClientBuilder(ApacheHttpTransport.java:143)
        at com.google.api.client.http.apache.v2.ApacheHttpTransport.newDefaultHttpClient(ApacheHttpTransport.java:116)
        at com.google.api.client.http.apache.v2.ApacheHttpTransport.<init>(ApacheHttpTransport.java:69)
        at com.google.cloud.tools.jib.http.FailoverHttpClient.getSecureHttpTransport(FailoverHttpClient.java:100)
        at com.google.cloud.tools.jib.http.FailoverHttpClient.getHttpTransport(FailoverHttpClient.java:329)
        at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:244)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:139)
        at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:115)
        at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:573)
        at com.google.cloud.tools.jib.registry.RegistryClient.pullManifest(RegistryClient.java:386)
        at com.google.cloud.tools.jib.registry.RegistryClient.pullManifest(RegistryClient.java:390)
        at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.pullBaseImage(PullBaseImageStep.java:194)
        at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:126)
        at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:56)
        at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
        at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:69)
        at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)


Next time:

Caused by: java.lang.NoClassDefFoundError: Could not initialize class org.apache.http.conn.ssl.SSLConnectionSocketFactory

 -- same --

--- a/build.gradle
+++ b/build.gradle
@@ -1,19 +1,19 @@
+// See http://dub.podval.org/2020/06/29/bintray-gradle-plugin.html for the reasons
+// this `buildscript` block is here and Bintray plugin is applied to the root project.
+buildscript {
+  dependencies {
+    classpath ("org.apache.httpcomponents:httpclient") {
+      version {
+        strictly '4.5.10'
+      }
+    }
+  }
+}
+
 plugins {
   id 'com.github.ben-manes.versions'
   id 'idea'
   id 'base'
-  // Note: Bintray plugin applied to a module reaches into other modules,
-  // which in some cases results in the following exception
-  //   Cannot cast object ... with class ... to class ...
-  // for some class within the Bintray plugin
-  // (class-loader issue; see http://forums.jfrog.org/Gradle-plugin-and-multi-projects-td7580521.html).
-  // For instance, this happens when:
-  // - `bintrayUpload` task for a specific module is run;
-  // - a module has both Bintray and Gradle Plugin Portal publishing plugin.
-  // It doesn't seem to be likely that Bintray will ever fix the problem;
-  // the workaround is: add Bintray plugin on the top level of the project *in addition* to the modules
-  // that actually use it, but mark it as not applied (there is no Bintray configuration on the top level,
-  // so plugin will fail with an NPE if applied):
   id 'com.jfrog.bintray' apply false
 }
 

--- a/library.gradle
+++ b/library.gradle
@@ -13,7 +13,7 @@ ext.configureLibrary = {
   final String gitHubRepository = "opentorah/opentorah"
   final String gitHubRepositoryUrl = "https://github.com/$gitHubRepository"
   final String codeUrl = "$gitHubRepositoryUrl/tree/master/$projectName"
-  final String orgName = 'Open Torah'
+  final String orgName = 'Open Torah Project'
   final String orgUrl = 'http://www.opentorah.org'
 
 //  logger.lifecycle(
@@ -91,8 +91,8 @@ ext.configureLibrary = {
               id = 'dub'
               name = 'Leonid Dubinsky'
               email = 'dub@opentorah.org'
-              organization = 'Open Torah Project'
-              organizationUrl = 'http://www.opentorah.org'
+              organization = orgName
+              organizationUrl = orgUrl
               timezone = '-5'
             }
           }

